src/js/components/PageSection/PageSection.js

```diff
@@ -0,0 +1,22 @@
+import React, { useContext } from 'react';
+import { Box } from '../Box';
+import { PageContext } from '../Page';
+import { PageSectionPropTypes } from './propTypes';
+
+const PageSection = ({ children, fullBackground, ...rest }) => {
+  const { ...pageContext } = useContext(PageContext);
+  const contents = (
+    <Box {...pageContext} {...rest}>
+      {children}
+    </Box>
+  );
+  if (fullBackground) {
+    return <Box background={fullBackground}>{contents}</Box>;
+  }
+  return contents;
+};
+
+PageSection.displayName = 'PageSection';
```

Comment:
I am wondering about the term "Section". HTML5 has `<section />`, https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section. But this doesn't seem to quite line up with it. Or, is the expectation that we would render this in the DOM as `<section />`? If not, I'm trying to think of another name that would convey the use. I think 80% of the time there will be only one, but sometimes there might be multiple for a Page. I was OK with `PageContent`, other ideas?

Reply:
Yeah I could see how the name PageSection could cause confusion with the section element. This was something I hadn't thought of. I agree that the majority of the time there will only be one "PageSection". I'm good with switching the name to PageContent

Reply:
I was actually wondering if the 80% case would be only one PageContent and whether we could allow a Page to have a single PageContent built in automatically, perhaps as a property that defaults to do that but you could turn it off, thoughts?

Reply:
I like this 👆 .

Reply:
Hmm actually I'm thinking it might make sense to create a Page with separate PageContent sections for the header, main content, and footer so the 80% case might not always be one PageContent within Page.

Reply:
I'm not sure that's an 80% case though. If I look at the GLCP designs, they don't have that background variation, which I think is the main driver of having multiple sections.

Reply:
Yeah that makes sense. I'll play around with having a PageContent built in automatically and a prop that you can switch this behavior on/off. I will push something up soon

Reply:
The marketplace main page has a full width banner area that has a background image (no margins). Then the main content area below the banner is background-dark although the content adheres to max width of what would be `wide`. Not that the background in this case is full even though the content is limited width. I'm not really seeing the point of the background width being restricted since I'd just do that via whatever my content layout box is. See https://www.figma.com/file/RFHKQmOmyn5pvY2nqa1WIs/VAS-Marketplace?node-id=3799%3A49344

Reply:
I pushed up some changes. I removed the `fullBackground` prop and now you can set `background={{ fill: 'horizontal' }}`. The Page has a PageContent section by default unless you set the `customizeContent` prop to true (still not quite sure about naming for the customizeContent prop).

With the design shared above that would be possible by setting `background={{ fill: 'horizontal', image: '...' }}` on the PageContent that is wrapping the "HPE GreenLake Marketplace" section. This would create the full background with the content maintaining its padding and restricted width.

I think it is useful to have the option to set the background of the PageContent without it horizontally filling the Page area. If you have a box within the PageContent and you set a background on that box its width won't be as wide as the PageContent since PageContent has some padding.